muxpi/nuc/flash.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+#
+# This script downloads, flashes the image and tests it on NUC. 
+# This script must be run on muxpi.
+#
+# Usage: 
+# $ ./flash.sh [DUP IP][.tar image][neptune/no parameters for minimal]
```

Comment:
The '[param]' means that it is an optional parameter. If you send an empty string it is not the same as optional.
```suggestion
# $ ./flash.sh DUP_IP IMAGE [neptune]
```